react-ui/src/components/LogInForm.js

```diff
@@ -55,15 +69,15 @@ class logInForm extends Component {
                   <input 
                   className ="InputFields" 
                   type="text" 
-                  name="user_id"
```

Comment:
I'd recommend keeping the terminology consistent with the Student and Worker objects. For user_ID and user_password: Check out the way students and service staff are stored in > MSCI342-Term-Project/server/models/data/Student.js / and MSCI342-Term-Project/server/models/data/Worker.js /